rlclientlib/schema/v2/MultiStepEvent.fbs

```diff
@@ -13,4 +13,9 @@ table MultiStepEvent {
     deferred_action:bool = false;
 }
 
+table EpisodeEvent {
+    episode_id:string;
```

Comment:
we already have this id in meta - do we need to duplicate it here?

Reply:
Technically this entire object is a dummy, we could leave it completely empty (if thats allowed). I just put the id in there since it might be useful at some point in time